<launch>
    <arg name="survive_active" default="true" />
    <arg name="survive_tracker" default="trackerD" />
    <arg name="side" default="y" />
    <arg name="map_tole" default="0.2" />
    <arg name="print_world_active" default="false" />
    <arg name="freq_" default="40" />
    <arg name="unit_" default="100" />
    <!-- <param name="side" value="b"/> global -->

    <group if="$(arg survive_active)">
        <node name="vive_world" pkg="vive" type="world" output="screen">
            <rosparam file="$(find vive)/param/vive_calibrate.yaml" command="load" /> <!-- tf_to_map -->
            <param name="tracker" type="string" value="$(arg survive_tracker)" />
            <param name="survive_prefix" type="string" value="survive_" />
            <param name="vel_topic_name" type="string" value="tracker_vel" />
            <param name="run_service_name" type="string" value="survive_world_is_running" />
            <param name="max_distance_bt_maps" type="double" value="$(arg map_tole)" />
            <param name="print" type="bool" value="$(arg print_world_active)" />
            <param name="freq" type="int" value="$(arg freq_)" />
            <param name="unit" type="int" value="$(arg unit_)" />
            <param name="side" type="string" value="$(arg side)" /> <!-- local -->
        </node>
    </group>

</launch>